modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/display/context/RenderLayoutStructureDisplayContext.java

```diff
@@ -748,7 +748,7 @@ public String getStyle(StyledLayoutStructureItem styledLayoutStructureItem)
 			styleSB.append("background-position: 50% 50%; background-repeat: ");
 			styleSB.append("no-repeat; background-size: cover; ");
 			styleSB.append("background-image: url(");
-			styleSB.append(backgroundImage);
+			styleSB.append(PortalUtil.getPathContext() + backgroundImage);
```

Comment:
Hey @rolandpakai, what do you think about moving this getPathContext call [to the return of `_getBackgroundImage ` ](https://github.com/liferay-frontend/liferay-portal/blob/7b93f45172e3f03683783a12c3b9d8254dd1177e/modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/display/context/RenderLayoutStructureDisplayContext.java#L1061)function?

Reply:
Hi @diegonvs No problem, we can move it there